_posts/2024-01-09-owasp-top10-with-abstract-examples-part-1.md

```diff
@@ -34,7 +34,7 @@ TrafiCom's report [The state of software security 2023](https://www.kyberturvall
 
 Not taking cyber security seriously can backfire in many ways. The average cost of a data breach in the United States between August 2019 and April 2020 was more than [$8.6 million](https://www.ibm.com/reports/data-breach). The same IBM report states that organizations that tested their accident prevention plan saved about $2 million on average. Why are the numbers so high? That's because data breaches don't just affect sales. These days, companies have to deal with regulatory fines, legal costs, operational downtime, and last but not least, reputational damage. In the worst case scenario, the organization's information can be encrypted, inaccessible to the organization, or eventually completely public.
 
-Someone could say that we are such a small operator that we are not interested in attackers. This can no longer be trusted, as attackers have discovered that small companies also have a smaller security budget. Because of this, they are also an easier target. There are also examples of this. Perhaps the most famous of these are [NotPetya and Maersk](https://en.wikipedia.org/wiki/2017_Ukraine_ransomware_attacks). The weakest link in the chain was found and the virus spread, partly by accident, to the information systems of a large company through the product of the smaller MeDoc accounting firm.
+Someone could say that we are such a small operator that we are not interested in attackers. This can no longer be trusted, as attackers have discovered that small companies also have a smaller security budget. Because of this, they are also easier targets. There are also examples of this. Perhaps the most famous of these are [NotPetya and Maersk](https://en.wikipedia.org/wiki/2017_Ukraine_ransomware_attacks). The weakest link in the chain was found and the virus spread, partly by accident, to the information systems of a large company through the product of the smaller MeDoc accounting firm.
```

Comment:
Pitäisikö tässä olla ennemmin "we are not interesting to attackers"?

Reply:
Joo. Voisin kyllä vaihtaa koko lauseen paremmaksi.

Reply:
"Someone could say that we are such a small player that attackers are not interested in us"